win/CS/HandBrakeWPF/ViewModels/SummaryViewModel.cs

```diff
@@ -230,7 +230,7 @@ public OutputFormat SelectedOutputFormat
                     this.NotifyOfPropertyChange(() => this.IsMkvOrWebm);
                     this.NotifyOfPropertyChange(() => this.IsIpodAtomVisible);
                     this.SetExtension(string.Format(".{0}", this.Task.OutputFormat.ToString().ToLower()));
-                    this.UpdateDisplayedInfo(); // output format may coreced to another due to container incompatibility
+                    this.UpdateDisplayedInfo(); // output format may be coerced to another due to container incompatibility
```

Comment:
I'm not certain about the main word fix, and I've added a word in an attempt to make the thought make sense.